src/material-experimental/mdc-slider/slider.ts

```diff
@@ -150,6 +154,11 @@ export class MatSliderVisualThumb implements AfterViewInit, OnDestroy {
     this._elementRef.nativeElement.removeEventListener('mouseleave', this._onMouseLeave);
   }
 
+  /** Used to append a class to indicate when the value indicator text is short. */
+  _isShortValue(): boolean {
+    return this.valueIndicatorText?.length <= 2;
```

Comment:
nit: it might be a good idea to initialize the `valueIndicatorText` to an empty string so that we don't have to null check it everywhere. That would also make it easier to inline the entire expression in the `host` binding.